modules/misc.go

```diff
@@ -80,6 +84,22 @@ Written using @gotgproto by @GIGADevs.
 	return dispatcher.EndGroups
 }
 
+func dotsave(ctx *ext.Context, u *ext.Update) error {
+	msg := u.EffectiveMessage
+	if msg.ReplyTo.ReplyToMsgID == 0 {
+		ctx.EditMessage(u.EffectiveChat().GetID(), &tg.MessagesEditMessageRequest{
+			ID:      u.EffectiveMessage.ID,
+			Message: "Reply to a message!",
+		})
+		return dispatcher.EndGroups
+	}
+	ctx.DeleteMessages(u.EffectiveChat().GetID(), []int{u.EffectiveMessage.ID})
```

Comment:
Using delete on successful command usage is not our code-wide standard, please use edit message and print a successfully saved statement.

Reply:
Ok